i18n/fr/docusaurus-plugin-content-docs/version-22.04/releases/centreon-commercial-extensions.md

```diff
@@ -168,6 +181,19 @@ The new MAP extension is now available in a full web version with a new server,
 
 ## Centreon MAP Legacy
 
+### 22.04.11
+
+Release date: `September 22, 2023`
+
+#### Bug fixes
+
+-	[Desktop] Update dependency. 
+-	[Server] Fixed the diagnostic protocol test.
+
```

Comment:
one missing (does not convey much infos though)
[Server] Fixed vulnarable property.